sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/AmqpExceptionHandler.java

```diff
@@ -24,7 +24,7 @@
      * @param exception The exception that caused the connection error.
      */
     void onConnectionError(Throwable exception) {
-        logger.warning("Connection exception encountered: " + exception.toString(), exception);
+        logger.warning("Connection exception encountered: " + exception, exception);
```

Comment:
Should this use a parameterized message?

```java
logger.warning("Connection exception encountered: {}", exception);
```

Right now this will likely log the exception message twice.